code/modules/vending/robotics.dm

```diff
@@ -8,7 +8,7 @@
 	products = list(/obj/item/clothing/suit/toggle/labcoat = 4,
 					/obj/item/clothing/under/rank/roboticist = 4,
 					/obj/item/stack/cable_coil = 4,
-					/obj/item/assembly/flash/handheld = 4,
+					/obj/item/assembly/flash/synthetic = 4,
```

Comment:
Needs // beat

Reply:
already have one here
https://github.com/Nopm/Beat-Station/pull/488/files#diff-8ba8b3c43278f126a9251f8983799f83R21

Reply:
All changes need it